### DISCUSSION TITLE
Problème de configuration de GitHub Pages pour un dossier spécifique

### DISCUSSION THREAD

**Discussion by Lucien-TFE** [community/community#123281] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
I'm having a problem configuring GitHub Pages for a specific folder in my GitHub repository. I tried to configure GitHub Pages to serve my site from the Jeu_Deren folder which contains an index.html file, but I can't seem to select this folder in the GitHub Pages configuration options.
Here are the details:
Repository name: Mon-jeu-phaser
Branch used: master
Desired folder for GitHub Pages: Jeu_Deren
Steps already taken:
I've checked that the Jeu_Deren folder contains an index.html file.
I've committed all recent changes, including folder renaming and file modifications.
I've waited and refreshed the GitHub Pages settings page several times, but the Jeu_Deren folder still doesn't appear in the drop-down menu for selecting the source.
Can you help me understand why this folder is not recognized as a valid source for GitHub Pages and how I can solve this problem?
Thank you for your assistance.
Best regards,
Leclerc Lucien

Translated with [DeepL.com](https://www.deepl.com/?utm_campaign=product&utm_source=web_translator&utm_medium=web&utm_content=copy_free_translation) (free version)

Bonjour,
Je rencontre un problème pour configurer GitHub Pages pour un dossier spécifique dans mon dépôt GitHub. J'ai essayé de configurer GitHub Pages pour servir mon site à partir du dossier Jeu_Deren qui contient un fichier index.html, mais je ne parviens pas à sélectionner ce dossier dans les options de configuration de GitHub Pages.
Voici les détails :
Nom du dépôt : Mon-jeu-phaser
Branch utilisée : master
Dossier souhaité pour GitHub Pages : Jeu_Deren
Démarches déjà entreprises :
J'ai vérifié que le dossier Jeu_Deren contient bien un fichier index.html.
J'ai commité tous les changements récents, y compris le renommage de dossiers et les modifications de fichiers.
J'ai attendu et rafraîchi plusieurs fois la page des paramètres de GitHub Pages, mais le dossier Jeu_Deren n'apparaît toujours pas dans le menu déroulant pour sélectionner la source.
Pouvez-vous m'aider à comprendre pourquoi ce dossier n'est pas reconnu comme une source valide pour GitHub Pages et comment je peux résoudre ce problème ?
Merci pour votre assistance.
Cordialement,
Leclerc Lucien

**Comment by yoannchaudet** [comment]
Hello,

The only folders you can select are either the root `/` or `/docs`. We don't allow arbitrary folders.

It is common for people to decide to use a dedicated branch for their site. That branch does not have to follow the same folder structure than the main branch in your repo.
